tests/fpga/symbol_mapping.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2019-2020 ETH Zurich and the DaCe authors. All rights reserved.
+
+# The scope of the test is to verify that code nested SDFGs with a unique name is generated only once
+# The nested SDFG compute vector addition on FPGA, with vectorization
```

Comment:
How do you check that it is only generated once? I can't see this in the test

Reply:
My fault, wrong comment